content/ko/training/_index.html

```diff
@@ -0,0 +1,108 @@
+---
+title: 훈련하기
+bigheader: 쿠버네티스 교육과 인증
+abstract: 교육 프로그램, 인증 그리고 파트너
+layout: basic
+cid: training
+class: training
+---
+
+<div class="padded blue-bg">
+  <div class="main-section two-thirds-centered white-text">
+    <center>
+      <h2>당신의 클라우드 네이티브 커리어를 구축하세요</h2>
+      <p>쿠버네티스는 클라우드 네이티브 무브먼트의 핵심입니다. 리눅스 재단과 교육 파트너의 교육 그리고 인증을 통해 당신의 커리어에 투자하고, 쿠버네티스를 배우며 클라우드 네이티브 프로젝트를 성공적으로 수행할 수 있습니다.</p>
+    </center>
+  </div>
+</div>
+
+<section>
+  <div class="main-section padded">
+    <center>
+      <h2>edX에서 무료 강좌 수강하기</h2>
+    </center>
+  <div class="col-container">
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>쿠버네티스 소개 <br> &nbsp;</b>
+        </h5>
+        <p>쿠버네티스를 배우고 싶습니까? 컨테이너화된 애플리케이션의 관리를 위한 강력한 시스템에 대한 심도있는 입문서를 얻으십시오.</p>
+        <br>
+        <a href="https://www.edx.org/course/introduction-to-kubernetes" target="_blank" class="button">강좌로 이동하기</a>
+      </center>
+    </div>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>클라우드 인프라 기술 소개</b>
+        </h5>
+        <p>오픈 소스의 리더인 리눅스 재단으로부터 직접 클라우드 기술을 구축하고 관리하는 기본사항을 배웁니다.</p>
```

Comment:
```suggestion
        <p>오픈 소스의 리더인 리눅스 재단으로부터 직접 클라우드를 구축하고 관리하는 기술의 기초를 배우세요.</p>
```

1. "클라우드 기술"이 아닌 "클라우드"를 구축하고 관리할 대상으로 보아야할 것 같습니다.
1. 페이지 내 다른 문장과의 일관성을 위해 청유형을 제안드립니다. 참고 바랍니다.